### PR TITLE
LITE-12104 Operator "tuple" for filtration by nested fields added

### PR DESCRIPTION
**Pull request by zzzevaka**

### CODE REVIEW

File: dj_rql/transformer.py [hunk 1]
```diff
@@ -72,13 +76,44 @@ class RQLToDjangoORMTransformer(BaseRQLTransformer):
...
+        elif (tree.data in self.NAMESPACE_FILLERS
+                and self._namespace
+                and self._namespace[-1] is None):
+            self._namespace[-1] = self._get_value(tree)
+
+    def _pop_namespace(self, tree):
+        if tree.data in self.NAMESPACE_PROVIDERS:
+            self._namespace.pop()
+        elif tree.data in self.NAMESPACE_ACTIVATORS:
+            self._active_namespace.pop()
```
Comment (maxipavlovic) [inline dj_rql/transformer.py:109]: Let's check if `_active_namespace` can be a number.

**Review by maxipavlovic — CHANGES_REQUESTED** [review]
PLGTM

File: dj_rql/filter_cls.py [hunk 2]
```diff
@@ -243,6 +243,12 @@ def build_q_for_filter(self, data):
         """
         filter_name, operator, str_value = data.filter_name, data.operator, data.str_value
         list_operator = data.list_operator
+        filter_basename, namespace = data.filter_basename, data.namespace
+
+        if namespace and filter_basename == RQL_SEARCH_PARAM:
+            raise RQLFilterLookupError(details={
+                'error': f'Filter "{filter_basename}" can be applied only on a global level.',
```
Comment (maxipavlovic) [inline dj_rql/filter_cls.py:250]: If we introduce this kind of message, I would suggest reusing it for all the stuff that follows the same patterns. Ping me to discuss that.

**Review by maxipavlovic — CHANGES_REQUESTED** [review]

File: dj_rql/filter_cls.py (hunk 2 above)
Comment (zzzevaka) [reply dj_rql/filter_cls.py]: Discussed. Grammar error fixed

**Review by maxipavlovic — APPROVED** [review]

**Pull request merged**